samples/opencl/FindOpenCL.cmake

```diff
@@ -0,0 +1,174 @@
+# Find OpenCL
```

Comment:
well, this might be possible. BTW, did you check OpenCV can be configured with CMake 3.x? I have locally CMake 2.8, so I did not try CMake 3.x yet.